libs/SalesforceSDK/src/com/salesforce/androidsdk/config/LoginServerManager.java

```diff
@@ -52,6 +52,9 @@
  * @author bhariharan
  */
 public class LoginServerManager {
+	// LiveData representation of the users current selected server.
+	public MutableLiveData<LoginServer> selectedServer
+			= new MutableLiveData<>(new LoginServer("Production", PRODUCTION_LOGIN_URL, false));
```

Comment:
If it ain't broke... wrap it in a new modern interface?  In all seriousness, I didn't feel like any changes to this class were actually necessary other than exposing a way to observe changes to the currently selected server.   

The only compelling reason I can think of to convert this class to Koltin right now is that `selectedServer` could be made internal.   Right now it could actually be set by the app, which isn't ideal.  

I might be able to restore the removed `private LoginServer selectedServer;` and use it as a backing value for `derivedStateOf` but I am not sure if that will work outside of a composable context.